### PR TITLE
New lighthouse audits post.

### PR DESCRIPTION
**Pull request by malchata**
Not totally done, yet. Two figures are missing, and I should write a concluding paragraph. But wanted to get this in sooner than later.

### CODE REVIEW

File: src/content/en/updates/2018/05/lighthouse.md [hunk 1]
```diff
@@ -234,6 +234,21 @@ loading performance by converting their GIFs to video. Additionally, if your
...
+
+If you're excited about these new audits, update Lighthouse and give them a try!
+
+* For Node users: Run `npm update lighthouse`, or `npm update lighthouse -g` if
+you installed Lighthouse globally.
+* For Chrome users:
+  * The Lighthouse extension should automatically update, but you can manually
+update it via `chrome://extensions`.
+  * In DevTools, The Audits panel will be shipping with `[version]` in Chrome
+`[version]`. You can check your version of Chrome via `chrome://version`. Chrome
```
Comment (addyosmani) [inline src/content/en/updates/2018/05/lighthouse.md]: @kaycebasques I'd love for us to coordinate with the blog post you wrote in the LH 3.0 round-up. Should we say 3.0 in here for consistency? I was considering if we should say "Many of the audits we have covered in this post have been landing in Lighthouse over the last few versions. The very latest audits can be found in Lighthouse 3.0." or something to that effect.

File: src/content/en/updates/2018/05/lighthouse.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New perf audits for preload, preconnect, GIFs, and more.
+
+{# wf_updated_on: 2018-05-04 #}
+{# wf_published_on: 2018-05-05 #}
```
Comment (addyosmani) [inline src/content/en/updates/2018/05/lighthouse.md]: Could we bump the updated and published on dates to the 8th please?

**Comment by addyosmani** [comment]
Kayce suggested it might be useful for us to try letting developers know which version of Lighthouse each of the audits landed in. If we have time, I'd say that would be a nice to have. 

If not, it's not the end of the world if we go with the current wording.

@malchata if you get a chance, could you confirm if https://developers.google.com/web/updates/2018/05/lighthouse will be the final URL (from your understanding) for this post?

**Comment by malchata** [comment]
Changed publish/update dates, @addyosmani.

**Comment by malchata** [comment]
I'm happy to add the versions, but does he have that info? Otherwise, it could take some time to dredge those up (unless there's a magic way to find that stuff out I wasn't aware of).

My local URL is http://localhost:8080/web/updates/2018/05/lighthouse, so yeah, I think the URL you specified will be the final.

**Comment by addyosmani** [comment]
Perfect. We'll land this either late tonight or early in the morning and aim to get a deploy done then. Thanks, @malchata!

**Pull request merged**